klasses/models.py

```diff
@@ -0,0 +1,40 @@
+"""Models for klasses"""
+from django.db.models.base import Model
+from django.db.models.fields import (
+    DateTimeField,
+    DecimalField,
+    TextField,
+    IntegerField,
+)
+from django.db.models.fields.related import ForeignKey
+
+
+class Bootcamp(Model):
+    """
+    A bootcamp
+    """
+    title = TextField()
+
+
+class Klass(Model):
+    """
+    A class within a bootcamp
+    """
+    bootcamp = ForeignKey(Bootcamp)
+    klass_id = IntegerField()
+    price = DecimalField(max_digits=20, decimal_places=2)
```

Comment:
If we keep the price we need to keep it in sync with the sum of the installments and this needs to be done either automatically (but I do not how right now) or manually (with the risk of have inconsistent data)